packages/orchestration/src/exos/cosmosOrchestrationAccount.js

```diff
@@ -363,6 +366,7 @@ export const prepareCosmosOrchestrationAccountKit = (
          */
         async getBalance(denom) {
           const { chainAddress, icqConnection } = this.state;
+          if (!icqConnection) throw Error('Queries not enabled.');
```

Comment:
This reads like something the developer could change.
```suggestion
          if (!icqConnection) throw Fail'Queries not enabled for chain ${chainAddress.chainId}`.`;
```

Reply:
Agree, updated